client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -343,6 +364,8 @@ class EmailProvidersComparison extends React.Component {
 					</Button>
 				</GSuiteNewUserList>
 			</FormFieldset>
+		) : (
+			<FormFieldset></FormFieldset>
```

Comment:
Ideally we should display a placeholder but since the Google Workspace card is collapsed by default when the page loads this seems like a good workaround.

Reply:
What about null? or just `<FormFieldset/>` ?

Reply:
Good point, I will change that.